### ISSUE TITLE
Fix Readme.md

### ISSUE THREAD

**Issue by xxcombat** [realm/SwiftLint#3964]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

**Need update README.md**

I have a problem with the install SwiwtLint on MacOs Monterey.
After installation OS this directory  **/usr/local/bin** is not.
If I run (need add 'sudo')
`sudo ln -s /opt/homebrew/bin/swiftlint /usr/local/bin`

I get a wrong symbolic link (file bin -> /opt/homebrew/bin/swiftlint)
```
sudo ln -s /opt/homebrew/bin/swiftlint /usr/local/bin
ls -l /usr/local
lrwxr-xr-x  1 root  wheel   27 Apr 30 21:36 bin -> /opt/homebrew/bin/swiftlint
```

**Valid command to create a symbolic link in **/usr/local/bin**** 

```
sudo mkdir /usr/local/bin
sudo ln -s /opt/homebrew/bin/swiftlint /usr/local/bin/
```

**Comment by SimplyDanny** [comment]
Thank you for bringing this up. The command now looks like

```bash
ln -s /opt/homebrew/bin/swiftlint /usr/local/bin/swiftlint
```

which should be sufficient to get the link right. If `/usr/local/bin` does not (yet) exist `ln` will report a clear error which the user can handle.
